dask_jobqueue/core.py

```diff
@@ -55,8 +55,11 @@
     job_script_prologue : list
         Other commands to add to script before launching worker.
     header_skip : list
-        Lines to skip in the header.
-        Header lines matching this text will be removed
+        Deprecated: use ``job_directives_skip`` instead. This parameter will be removed in a future version.
+    job_directives_skip : list
+        Directives to skip in the generated job script header.
+        Directives lines matching this text will be removed.
```

Comment:
"matching" is a bit unclear to me. It is implemented as a substring and its a string so IMHO `Directives lines containing the specified strings will not be submitted` is a tiny bit more clear to me. But it might be a matter of taste.